tests/README.md

```diff
@@ -434,6 +434,13 @@ the test driver machine.
 
         $ cd kafka
         $ ./vagrant/aws/aws-init.sh
+
+* Add your AWS keys and load them (https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/):
```

Comment:
This might be an additional incremental improvement step for this README. Up to now, this script was never mentioned and thus, I figured out a solution that worked... To be honest, don't have time to figure out how to use this script right now to update the README accordingly...